style.css

```diff
@@ -44,6 +44,11 @@ Tags: one-column, accessibility-ready, custom-menu, featured-images, footer-widg
 	--content-width:         1080px;
 	--featured-height:       450px;
 	--border-radius:         3px;
+
+  --form-background-color:       #ffffff;
+  --form-border-color:           #cacad8;
+  --form-border-color-focused:   #34343a;
+  --form-text-color:             #000000;
```

Comment:
Looks like we use tabs in style.css? 😱 

We might want to fix that later on, but unfortunately I must ask you to tab-ify all of your spaces in style.css

Reply:
I re-add the missing evil characters :)